### ISSUE TITLE
UI: Transaction history / details + menu display

### ISSUE THREAD

**Issue by swedishfrenchpress** [cashubtc/eNuts#154]
The current combination of background and text for many settings options in the app does not meet the WCAG AAA level, the highest standard. Currently, it only meets the AA standard, which is the minimum requirement for legal compliance.

One concern I have is the color choices of green and red for the incoming and outgoing satoshi amounts in the transaction history. Given that red/green color blindness is the most prevalent, using this color combination without a "+" or "-" indicator could be confusing for users. This indicator would clearly signify the direction of funds, making it best practice.

I also have feedback on the display of line items in the settings. The current UI encloses the text in ovals, which seem to add unnecessary visual elements. Some suggestions:

**Transaction History**
![s1](https://github.com/cashubtc/eNuts/assets/78821053/abfd2d6e-1b63-46d5-8b13-c506308ddcdd)
- Streamline the settings display with underlined rows.
- Use left and right arrows to indicate incoming and outgoing transactions.
- Add "+" and "-" to transactions for clarity on direction.
- Displaying dates under each transaction seems redundant; consider grouping transactions under a date header.
- Use a space, not a comma, as the thousands separator for better legibility.
- Replaced thunderbolt icon with sats denomination

**Payment Details**
![s2](https://github.com/cashubtc/eNuts/assets/78821053/885cb9a6-41dc-4cf8-a15d-0bfebc35a1c4)
- Add a "Transaction Details" header.
- Remove the comma between the date and time.
- Renames satoshi to sats

**Miscellaneous Thoughts**
I'm open to all any feedback here. I know this design is suggestion a pretty large change from the existing UI and the last thing I want do is create extra work for the sake of work, but I do believe that some of these suggestions would help.

**Comment by KKA11010** [comment]
Converting this into a discussion
